src/extractFromFilesFixture/LogoutView.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+
+import i18n from 'i18n';
```

Comment:
Maybe we should write some specific TypeScript in this file?

Reply:
yes you right :)

Reply:
Done @oliviertassinari